NuRadioMC/test/SingleEvents/test_build.sh

```diff
@@ -8,5 +8,5 @@ NuRadioMC/test/SingleEvents/T04validate_allmost_equal.py NuRadioMC/test/SingleEv
 
 NuRadioMC/test/SingleEvents/T05validate_nur_file.py NuRadioMC/test/SingleEvents/1e18_output.nur NuRadioMC/test/SingleEvents/1e18_output_reference.nur
 
-NuRadioMC/test/SingleEvents/T02RunSimulation.py NuRadioMC/test/SingleEvents/1e18_output_noise_reference.hdf5 NuRadioMC/test/SingleEvents/surface_station_1GHz.json NuRadioMC/test/SingleEvents/config_noise.yaml NuRadioMC/test/SingleEvents/1e18_output_noise.hdf5
```

Comment:
Does this change make sense? Using a different input file all of a sudden?

Reply:
Yes, this is actually the better choice. Before I used the noiseless reference to produce the noise input. Which I then used as input again. This was possible because we couldn't reproduce runs so well but with the improvements in the random generators this is no longer required.